src/server/service/slackbot.js

```diff
@@ -265,6 +265,7 @@ class SlackBotService extends S2sMessageHandlable {
             this.generateInputSectionBlock('path', 'Path', 'path_input', false, '/path'),
             this.generateInputSectionBlock('contents', 'Contents', 'contents_input', true, 'Input with Markdown...'),
           ],
+          private_metadata: body.channel_id,
```

Comment:
ここには他のものが入ってくることがあるのでオブジェクト形式てセットした方が良さそうです
```suggestion
          private_metadata: { channelId: body.channel_id}
```